Create spectator-contest.md (#45)
WISVCH/nwerc-2022: content/news/spectator-contest.md

---

> title: "Livestream URLs"

Reviewer: ```suggestion
title: "Spectator contest"
```

date: 2022-11-24T09:00:00+01:00
draft: false
---

> There will be a spectator contest hosted on [spectator.nwerc.eu](http://spectator.nwerc.eu).

Reviewer: ```suggestion
There will be a spectator contest hosted on [spectator.nwerc.eu](https://spectator.nwerc.eu).
```

Here you can self-register your team!

> The contest will start at 10:30 on Sunday (November 27 2022). This is half an our later then the real contest.

Reviewer: ```suggestion
The contest will start at 10:30 CET on Sunday (November 27th, 2022). This is half an our later than the start of the real contest.
```
